test/features/my_academics.feature

```diff
@@ -0,0 +1,7 @@
+Feature: My Academics Enrollment Page
+  User should be able to click on schedule of classes,add,drop,swap,edit classes
```

Comment:
@davidlimcheng @redconfetti 'User' is not very descriptive. I have replaced it with 'As a student (with no holds on my account), I should be ...'. I have also changed the advisor feature slightly so that they are written in the same way.